content-services/content-sources-backend
# Fixes 1672: support multiple gpg keys

## Summary
Previously, if there were multiple keys in a gpg key file, only the first key was read. Now multiple keys can be read.

## Testing steps
1. In UI, create this repo: https://dl.google.com/linux/chrome/rpm/stable/x86_64
2. With this gpg key: https://dl.google.com/linux/linux_signing_key.pub
3. With metadata verification set to true
4. Should not return a validation error. Previously, would return the error `Error validating signature: openpgp: signature made by unknown entity. Is this the correct GPG Key?`
